pkg/operator/etcdcertsigner/etcdcertsignercontroller.go

```diff
@@ -305,6 +305,7 @@ func (c *EtcdCertSignerController) createSecretForNode(node *corev1.Node) error
 		"etcd.openshift-etcd.svc.cluster.local",
 		"*." + etcdDiscoveryDomain,
 		"127.0.0.1",
+		"::1",
```

Comment:
in theory, this is the same as below.

Reply:
Yeah handshake does not seem smart enough to understand long form.